packages/components/Icon/src/Icon.types.ts

```diff
@@ -0,0 +1,35 @@
+import { SvgProps } from 'react-native-svg';
+export const iconName = 'Icon';
+
+export interface IFontIconProps
+{
+  fontFamily?: string;
+  fontSrcFile?: string;
+  codepoint: number;
+  fontSize?: number;
+  color?: string;
+}
+
+export interface ISvgIconProps
+{
+  uri?: string;
+  src?: React.FC<SvgProps>;
+  viewBox?: string;
+  color?: string;
+}
+
+export interface IRasterImageIconProps
+{
+  src: object;
```

Comment:
Should this be typed to match ImageProps (src: ImageProps['src']? Or just be ImageProps?